docs/style-guide.rst

```diff
@@ -112,7 +112,8 @@ No::
         }
     }
 
-Maximum Line Length
+.. _maximum_line_length:
```

Comment:
Not sure if that is the reason, but in other parts we use dashes and not underscores. That seems to be working `.. _maximum-line-length` and the reference must be changed too.

Also please keep an empty new line after it.

Reply:
I added the newline; but, in [the file I just checked](https://github.com/OTTTO/solidity/edit/develop/docs/security-considerations.rst) in the style guide I saw underlines.  
Also I noticed that in the GitHub preview, :ref:'words <reference>' [is always present](https://github.com/OTTTO/solidity/blob/develop/docs/security-considerations.rst); but, on [the official docs](http://solidity.readthedocs.io/en/develop/security-considerations.html) it becomes an internal link
So, if you agree with the changes we should be able to merge.

Reply:
I personally don't mind which as long as the documentation compiles properly. Have you checked it? Just run `make html` in the `docs` directory.

Reply:
it works 👍